static/app/views/projectDetail/charts/projectBaseSessionsChart.tsx

```diff
@@ -84,6 +87,7 @@ function ProjectBaseSessionsChart({
                     end={end}
                     projects={projects}
                     environments={environments}
+                    queryExtra={{query}}
```

Comment:
We do not need `queryExtra` here, because it does not control what gets fetched from API, but it only adds those extra queries to the URL when you click on a release line and navigate to the release details page.